README.md

```diff
@@ -288,6 +288,7 @@ See the available options in the table below.
 | minRadiusCircleMarker | `null`                                | Set the min radius of a `CircleMarker` when editable is active.                                                                                                                                                 |
 | maxRadiusCircleMarker | `null`                                | Set the max radius of a `CircleMarker` when editable is active.                                                                                                                                                 |
 | editable              | `false`                               | Makes a `CircleMarker` editable like a `Circle`.                                                                                                                                                                |
+| radiusEdit            | `true`                                | Enables radius editing after drawing a `Circle`
```

Comment:
I'm thinking of naming it `radiusEditCircle` then we can have in future `radiusEditCircleMarker` with a different value.